pkg/config/dynamicconfig.go

```diff
@@ -235,13 +215,23 @@ func (cd *ConfigData) load(cm v1.ConfigMap) {
 		cd.excludeGroupRole = newExcludeGroupRoles
 	}
 
-	excludeUsernames := parseRbac(excludeUsername)
-	if reflect.DeepEqual(excludeUsernames, cd.excludeUsername) {
-		logger.V(4).Info("excludeGroupRole did not change")
-	} else {
-		logger.V(2).Info("Updated resource excludeUsernames", "oldExcludeUsername", cd.excludeUsername, "newExcludeUsername", excludeUsernames)
-		// update filters
-		cd.excludeUsername = excludeUsernames
+	// get resource filters
+	excludeUsername, ok := cm.Data["excludeUsername"]
+	if !ok  {
+		logger.V(4).Info("configuration: No excludeUsername defined in ConfigMap")
+	}
+
+	if excludeUsername == "" {
```

Comment:
Here is also a duplicate check.

Reply:
i know that but we are using these check before my changes also. And i thought you need dual check for these variables

Reply:
No, let's remove it.